Dockerfile-devcore-centos-6

```diff
@@ -24,7 +24,7 @@ RUN wget "http://mirror.metrocast.net/apache/maven/maven-3/3.3.9/binaries/apache
 RUN wget "https://www.apache.org/dist/ant/binaries/apache-ant-1.9.13-bin.tar.gz"                          -O - | tar --no-same-owner -xzf - -C /home/build/.zm-dev-tools/
 
 # Use a newer git
-RUN yum install -y zlib-devel gettext
+RUN yum install -y zlib-devel gettext curl-devel
```

Comment:
Fixes issue "Unable to find remote helper for 'https'" when cloning a repo
https://app.circleci.com/pipelines/github/ZimbraOS/zm-modern-zimlets/45/workflows/bf7bc371-9322-430c-b5e6-1bc49439a400/jobs/264